domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/eventcasefield/EventCaseFieldCasePaymentHistoryViewerCaseFieldValidator.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.ccd.definition.store.domain.validation.eventcasefield;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationResult;
+import uk.gov.hmcts.ccd.definition.store.repository.DisplayContext;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.EventCaseFieldEntity;
+
+@Component
+public class EventCaseFieldCasePaymentHistoryViewerCaseFieldValidator implements EventCaseFieldEntityValidator {
+
+    @Override
+    public ValidationResult validate(EventCaseFieldEntity eventCaseFieldEntity,
+                                     EventCaseFieldEntityValidationContext eventCaseFieldEntityValidationContext) {
+        ValidationResult validationResult = new ValidationResult();
+
+        if ("CasePaymentHistoryViewer".equals(eventCaseFieldEntity.getCaseField().getFieldType().getReference())
```

Comment:
any way we can avoid hardcoding the filed type id here? 
Perhaps accessing it via FieldTypeUtils.BASE_CASE_PAYMENT_HISTORY_VIEWER or similar?